src/expressions/array-expr.md

```diff
@@ -50,4 +61,14 @@ let y = (["a", "b"])[n];  // panics
 
 let arr = ["a", "b"];
 arr[10];                  // panics
+
+# let b = [[1, 0, 0], [0, 1, 0], [0, 0, 1]];
```

Comment:
Since we show that `arr[10]` panics, we should probably show this array being defined or else somebody may wonder why `b[1][2]` doesn't panic.

Reply:
Done. Also moved it up so it isn't grouped alongside the panic examples.